book/05-distributed-git/sections/contributing.asc

```diff
@@ -143,7 +143,12 @@ To jessica@githost:simplegit.git
    1edee6b..fbff5bc  master -> master
 ----
 
-John tries to push his change up, too:
+The last line of the output above shows a useful return message from the push operation.
+The basic format is `<oldref>..<newref> fromref \-> toref`, where `oldref` means the old reference, `newref` means the new reference, `fromref` is the name of the local reference being pushed, and `toref` is the name of the remote reference being updated.
+You'll see similar output like this below in the discussions, so having a basic idea of the meaning will help in understanding the various states of the repositories.
+More details are available in the documentation for https://git-scm.com/docs/git-push[git push].
```

Comment:
Can you change the link text to "git-push"? That's how you'd get the man page.

Reply:
@ben .  Your requested change has been made.  Thanks.